lib/matterStacks.ts

```diff
@@ -87,14 +104,10 @@ export class MatterStack extends Stack {
         }
         else {
             // Create PAI
-            let prodIds = new CfnParameter(this, "productIds", {
+            let prodIdsInput = new CfnParameter(this, "productIds", {
                 type: "List<String>",
                 description: "A list of product IDs to create first PAIs. These must be 4-digit hex values."
             })?.valueAsList;
-            const vendorId = new CfnParameter(this, "vendorId", {
```

Comment:
I think we need to update the README to remove the vendorId from the sample commands when creating a PAI